src/test/java/seedu/address/logic/parser/DeleteCommandParserTest.java

```diff
@@ -23,10 +24,17 @@ public class DeleteCommandParserTest {
     @Test
     public void parse_validArgs_returnsDeleteCommand() {
         assertParseSuccess(parser, "1", new DeleteCommand(INDEX_FIRST_PERSON));
+
```

Comment:
Good work on adding the testing for the updated `delete` command to handle parsing the names for deletion 👍